core/test/gather/driver/execution-context-test.js

```diff
@@ -41,7 +41,8 @@ describe('ExecutionContext', () => {
   it('should clear context on frame navigations', async () => {
     const executionContext = new ExecutionContext(sessionMock);
 
-    const frameListener = sessionMock.on.mock.calls.find(call => call[0] === 'Page.frameNavigated');
+    const frameListener = sessionMock.on.mock.calls
+      .find(call => call[0] === 'Page.frameNavigated') ?? [];
```

Comment:
I don't understand this change. The test needs to ensure that `frameListener` is defined, but it's impossible to be `undefined` now that you added the `?? []`.